Examples/en/TypeScript/Language/Soundness.ts

```diff
@@ -0,0 +1,109 @@
+// Without a background in type theory, you're unlikely
+// to be familiar with the idea of a type system being "sound"
+
+// Soundness is the idea that the compiler can make guarantees
+// about what types are available at runtime, and not just
```

Comment:
Maybe: _Soundness is the idea that the compiler can make guarantees about the type a value has a runtime, and not just during compilation._

<sup>Apologies if you aren't taking feedback atm</sup>

Reply:
I am - this is a solid improvement 👍